activerecord/lib/active_record/migration.rb

```diff
@@ -307,9 +307,9 @@ def initialize
   #
   # == Reversible Migrations
   #
-  # Starting with Rails 3.1, you will be able to define reversible migrations.
+  # Starting with Rails 3.1, you are able to define reversible migrations.
```

Comment:
Agree with @robin850. We can just say that this is possible, no need for the version back reference.